pkg/services/rpcsrv/server.go

```diff
@@ -282,11 +279,16 @@ func New(chain Ledger, conf config.RPC, coreServer *network.Server,
 	if orc != nil {
 		oracleWrapped.Store(&orc)
 	}
+	var wsOriginChecker func(*http.Request) bool
+	if conf.EnableCORSWorkaround {
```

Comment:
Do we need to update documentation for this option? There's only a couple of words about CORS, and this code is related to origin check.

Reply:
Updated!